Header header
string msg_type
string frame_id
uint8[] dest_addr
string command
